### DISCUSSION TITLE
CIS 8

### DISCUSSION THREAD

**Discussion by n1co38** [intuitem/ciso-assistant-community#198] [Ideas]
Hello everyone!

Is it possible to implement CIS 8 Control?

Thank you!

**Comment by eric-intuitem** [comment]
Hi @n1co38 please have a look in the tools directory, there is a script called cis/convert_cis.py that allows you to convert the official Excel file published by the CIS into your own library, which you can then import easily. This is necessary for copyright reasons.

**Comment by n1co38** [comment]
Bonjour,

Je viens de faire le test, j'ai le message d'erreur suivant :
 ./convert_cis.sh cis-controls-v8.xlsx
./convert_cis.sh: line 8: python: command not found
./convert_cis.sh: line 9: python: command not found
Resulting file is available at cis-controls-v8.yaml

Merci,
Nicolas Maurino

Le ven. 5 avr. 2024 à 13:34, eric-intuitem ***@***.***> a
écrit :
